### PR TITLE
Add Amazon Aurora PostgreSQL to the list of tested databases

### LINKED ISSUE #27048: Add Amazon Aurora PostgreSQL to the list of tested databases
### Description

In Keycloak, we now run nightly tests again Aurora AWS PostgreSQL since #26730  

This should be reflected in the documentation

### Discussion

_No response_

### Motivation

People should know that this is now supported. 

### Details

_No response_

### PR DESCRIPTION
**Pull request by ahus1**
Closes #27048

### CODE REVIEW

**Comment by ahus1** [comment]
@ryanemerson - this is a first draft to update the documentation. The build is still running in my repo. Maybe you can still have a first look at the docs I wrote. Thanks!

File: pom.xml [hunk 1]
```diff
@@ -160,6 +160,8 @@
         <mysql.version>8.0</mysql.version>
         <mysql-jdbc.version>8.3.0</mysql-jdbc.version>
         <postgresql.version>15</postgresql.version>
+        <autora-postgresql.version>15.3</autora-postgresql.version>
```
Comment (ryanemerson) [inline pom.xml:163]: `aurora-postgresql`?

Comment (ahus1) [reply pom.xml]: Thanks, fixed in next commit I just pushed. Builds looked good, before, waiting for another green build before marking it ready.

**Comment by ahus1** [comment]
There is a green run in my personal repo running against Aurora https://github.com/ahus1/keycloak/actions/runs/7919817005

Ready to review! @ryanemerson - please review when you have the time. Thanks!

**Review by ryanemerson — APPROVED** [review]

**Review by mhajas — APPROVED** [review]
Thanks @ahus1 @ryanemerson

**Pull request merged**